yarn.lock

```diff
@@ -19336,6 +19338,43 @@ fsevents@~2.1.2:
   languageName: node
   linkType: hard
 
+"webpack@npm:^5.10.2":
```

Comment:
Can we just align this with the version in `packages/apps` (5.10.1, just so there is one version the lockfile, instead of 2 as installed atm)